### ISSUE TITLE
Make the Field of View unlimited

### ISSUE THREAD

**Issue by Dyhr** [RE-SS3D/Client#36]

**Comment by MilaBits** [comment]
What does this mean exactly? 

Should it extend outside the camera's view?

**Comment by Dyhr** [comment]
Yeah, I guess. Currently it makes it very claustrophobic that you can't see 5 meters in front of you.
I might argue that we could compromise to just make it a longer range so people with ultra wide screens don't have a dramatically different view than everyone else, but people with normal monitors won't see the edge.

**Comment by MilaBits** [comment]
We can also rework colors, and potentially make the fog a bit transparent
